### PR TITLE
feat!: Create Gamut-kit

### PR DESCRIPTION
**Pull request by jakemhiller**
### Overview

Created gamut-kit package to simplify version upgrades across all gamut packages

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@1.0.1-alpha.4d8dae.0<br/></pre>

File: packages/gamut-kit/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# GAMUT KIT
```
Comment (codecaaron) [inline packages/gamut-kit/README.md:1]:
```suggestion
# GAMUT KIT (& KABOODLE)
```

**Review by codecaaron — APPROVED** [review]
<3

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.0.2-alpha.13ae2e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60414b135e61cd08af667fda--gamut-preview.netlify.app>https://60414b135e61cd08af667fda--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60414b135e61cd08af667fda>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.0.2-alpha.6e53e7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60414c7a951db20f8382dff7--gamut-preview.netlify.app>https://60414c7a951db20f8382dff7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60414c7a951db20f8382dff7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.0.2-alpha.4f2f71.0<br/></pre>

**Review by dreamwasp — APPROVED** [review]
yesssssssssssssssssss

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60414e7224314d11673dc32f--gamut-preview.netlify.app>https://60414e7224314d11673dc32f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60414e7224314d11673dc32f>Deploy Logs</a></p>

**Review by JoshuaKGoldberg — APPROVED** [review]
Lol, kaboodle. Let's do it!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.0.2-alpha.3e6c9c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6041561e1dd2e418ba08e73d--gamut-preview.netlify.app>https://6041561e1dd2e418ba08e73d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6041561e1dd2e418ba08e73d>Deploy Logs</a></p>

**Pull request merged**